### DISCUSSION TITLE
Full Page Screenshots

### DISCUSSION THREAD

**Discussion by abey** [pointfreeco/swift-snapshot-testing#488] [Ideas]
Full Page screenshots will be very useful for testing scrollviews/contentviews/etc. 

I'd love to see this integrated in the framework.

**Comment by urbanp11** [comment]
Like this?

![testFullPage 1](https://user-images.githubusercontent.com/11543745/166076395-58867dab-0810-4164-8d13-6a7952493a8f.png)
![testFullPage 2](https://user-images.githubusercontent.com/11543745/166076401-4917da44-9bb3-4f14-ab66-595ea297a4b1.png)

All you need is `assertSnapshot(matching: fullPageView, as: .image(layout: .sizeThatFits, traits: UITraitCollection(userInterfaceStyle: .light)))` and some changes in library so it can work also for `preferredContentSizeCategory`.
